updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding openshift upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} 4 upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+== Cluster upgrade duration
+
+Historical upgrade duration of similar clusters provides the best estimate for the future upgrades. However, if the historical data is not available, you can refer the below table to estimate the time duration of cluster update.
+
+The following table summarizes the approximate time duration of cluster update components:
+
+[options="header"]
+|===
+|Component Name |Time Duration (in minutes)
+
+|Cluster Version Operator (CVO)
+|45-120
+
+|Machine Config Operator (MCO)
+|5 per node
+|===
+
+=== Cluster Version Operator target upgrade payload deployment
+The Cluster Version Operator (CVO) unpacks the target upgrade release image and applies to the cluster. This process might take somewhere between 45 to 120 minutes.
```

Comment:
nit: maybe "retrieves" instead of "unpacks"?  Or something that reminds folks that there is network traffic between the cluster and the (mirror) registry, because for some clusters that network bandwidth may be a significant factor.

Reply:
@sdodson Included your suggestion. Thanks